### DISCUSSION TITLE
Can I insert my dissolution data into PK-Sim?

### DISCUSSION THREAD

**Discussion by Esther806** [Open-Systems-Pharmacology/Forum#835] [Questions & Problems] [answered]

**Comment by msevestre** [comment]
@Esther806 May I suggest explaining what you are trying to do in the description of your entry so that one can help you?

**Comment by Esther806** [comment]
I am going to do dissolution curve of acyclovir using PK-Sim

**Comment by AndreDlm** [comment, marked as answer]
If you intend to directly use the dissolution curve as is, you can do this via the table formulation type:

![Screenshot 2020-09-14 072720](https://user-images.githubusercontent.com/26541603/93047041-c70a0180-f65b-11ea-8961-0261f502a98e.png)

Is this what you were looking for?
